.ci.yaml

```diff
@@ -205,6 +205,8 @@ targets:
     properties:
       release_build: "true"
       config_name: linux_fuchsia
+    drone_dimensions:
```

Comment:
Could you please add a comment why we need this field, to avoid future removal mistakenly?

Reply:
Sure, added a comment